core/spec/models/spree/concerns/user_address_book_spec.rb

```diff
@@ -398,5 +476,144 @@ module Spree
         expect(user.ship_address).to eq(address)
       end
     end
+
+    describe "#bill_address=" do
+      let!(:user) { create(:user) }
+      let!(:address) { create(:address) }
+
+      # https://github.com/solidusio/solidus/issues/1241
+      it "resets the association and persists" do
```

Comment:
I think that adding exactly what gets persisted may add clarity to the message

Reply:
Done